src/magneticField/turbulentField/PlaneWaveTurbulence.cpp

```diff
@@ -100,17 +74,10 @@ PlaneWaveTurbulence::PlaneWaveTurbulence(const TurbulenceSpectrum &spectrum,
 	KISS_LOG_INFO << "PlaneWaveTurbulence: Using SIMD TD13 implementation"
 	              << std::endl;
 
```

Comment:
What would happen when the code is compiled with SIMD support but run on a CPU that does not support SIMD?
That's of course nothing that should be done but you never know.

My question is, will this lead to some unexpected behavior or just break the code. In the latter case I would be fine with not replacing the check.

Reply:
From my experience (and a brief google search), the program should crash with an error that says something about invalid instructions. I can do some more in-depth research and see if this behavior can be relied upon.

Reply:
I think we can live we such a behavior then. However, additional research is always nice but I wouldn't put as a high priority, when you have other things to do.

Reply:
Okay! I've written it down at low priority.